firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -761,6 +762,23 @@ class BrowserCoordinator: BaseCoordinator,
             performIfCoordinatorRootVCIsPresented(SettingsCoordinator.self) {
                 didFinishSettings(from: $0)
             }
+        case .syncMenuOpened:
+            guard uuid != windowUUID else { return }
+            let browserPresentedVC = router.navigationController.presentedViewController
+            if let navVCs = (browserPresentedVC as? UINavigationController)?.viewControllers,
+               navVCs.contains(where: {
+                   $0 is FirefoxAccountSignInViewController || $0 is SyncContentSettingsViewController
+               }) {
+                router.dismiss(animated: true, completion: nil)
```

Comment:
do we also want to remove any coordinators here?

Reply:
No these VCs don't have coordinators associated with them when they're presented (afaics) so there's nothing to remove in this case.

Reply:
sounds good, thanks for confirming!